example/web-avatar-client/test/mmlParsing.test.ts

```diff
@@ -0,0 +1,105 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import collectionData from "../src/collection.json";
+import { findAssetsInCollection } from "../src/findAssetsInCollection";
+import { parseMMLDescription } from "../src/parseMMLDescription";
+
+import {
+  threeNestedMCharacters,
+  threeRogueMModels,
+  twoInvalidlyWrappedMModel,
+  twoInvalidlyWrappedMModelOnInvalidMCharacterWith2ValidMModels,
+  twoRedundantMCharacterswithThreeMModelsEach,
+  validMCharacter,
+  validMCharacterWithOneInvalidMModel,
+  validMCharacterWithTwoInvalidMModels,
+} from "./test-data";
+import { extractNumberFromErrorMessage } from "./test-utils";
+
+describe("WebAvatarClient Test Utils", () => {
```

Comment:
These tests could include the parsed character data as well as the errors

Reply:
done